daprdocs/content/en/contributing/contributing-docs.md

```diff
@@ -36,6 +35,14 @@ These conventions should be followed throughout all Dapr documentation to ensure
 - **Assume a new developer audience** - Some obvious steps can seem hard. E.g. Now set an environment variable Dapr to a value X. It is better to give the reader the explicit command to do this, rather than having them figure this out.
 - **Use present tense** - Avoid sentences like "this command will install redis", which implies the action is in the future. Instead use "This command installs redis" which is in the present tense.
 
+## Diagrams and images
+It is strongly encouraged to create diagrams and images where ever possible for documentation pages. All diagrams are kept in a [Dapr Diagram Deck](/images/dapr-diagrams.pptx) which has guidance on style and icons. The diagram images are saved as PNG files into the [images folder](/images). 
+Diagrams should be;
+- Saved as PNG files with a high resolution
+- Named using the convention of a concept or building block so that they are grouped. For example `service-invocation-overview.png`. Also see Images guidance section below.
+- Added to the correct section in the `Dapr-Diagrams.pptx` deck so that they can be amended and updated.
+
```

Comment:
{{< button text="Download the Dapr Diagrams Deck" link="/images/dapr-diagrams.pptx" >}}